### PR TITLE
Misc. Changes to DDR4 OC Guide

### PR DESCRIPTION
**Pull request by Arshia1381**
Changes:
> Added information on ODT including a source, with info about behavior
> Updated some text to include Ryzen 5000
> Added tRTP to list of timings forced even by GDM
> Updated info on tCWL to include AMD systems
> Added update to info about VDDP

### CODE REVIEW

File: DDR4 OC Guide.md [hunk 1]
```diff
@@ -480,6 +480,7 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
    * On Intel, changing tCWL will affect tWRRD_dg/sg and thus tWTR_S/L. If you lower tCWL by 1 you need to lower tWRRD_dg/sg by 1 to keep the same tWTR values. Note that this might also affect tWR per the relationship described earlier.
    * <sup>1</sup>Some motherboards don't play nice with odd tCWL. For example, I'm stable at 4000 15-19-19 tCWL 14, yet tCWL 15 doesn't even POST. Another user has had similar experiences. Some motherboards may seem fine but have issues with it at higher frequencies (Asus). Manually setting tCWL equal to tCL if tCL is even or one below if tCL is uneven should alleviate this (eg. if tCL = 18 try tCWL = 18 or 16, if tCL = 17 try tCWL = 16).
    * The extreme preset is not the minimum floor in this case. tRTP can go as low as 6, while tWTRS/L can go as low as 1 6. Some boards are fine doing tCWL as low as tCL - 6. Keep in mind that this *will* increase the load on your memory controller.
+   * On AMD, the existence of Gear Down Mode may lead to odd values of tCWL not working. To run tCWL at an odd value, Gear Down Mode must be turned off. On Ryzen 3000 and 5000, tCWL can often be set to tCL - 2 but is know to require higher tWRRD.
```
Comment (integralfx) [inline DDR4 OC Guide.md:483]:
```suggestion
   * On AMD, tCWL can often be set to tCL - 2 but is known to require higher tWRRD.
```
GDM is already mentioned in the AMD section

File: DDR4 OC Guide.md [hunk 2]
```diff
@@ -339,7 +339,7 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
     > Meanwhile with 2 DPC SR config there is no issue in reaching 1866MHz FCLK/UCLK.  
 [~ The Stilt](https://www.overclock.net/forum/10-amd-cpus/1728758-strictly-technical-matisse-not-really-26.html#post28052342)
 * tRCD is split into tRCDRD (read) and tRCDWR (write). Usually, tRCDWR can go lower than tRCDRD, but I haven't noticed any performance improvements from lowering tRCDWR. It's best to keep them the same.
-* Geardown mode (GDM) is automatically enabled above 2666MHz, which forces even tCL, even tCWL and CR 1T. If you want to run odd tCL, disable GDM. If you're unstable try running CR 2T, but that may negate the performance gain from dropping tCL.
+* Geardown mode (GDM) is automatically enabled above 2666MHz, which forces even tCL, even tCWL, even tRTP and other even timings and CR 1T. If you want to run odd tCL, disable GDM. If you're unstable try running CR 2T, but that may negate the performance gain from dropping tCL.
```
Comment (integralfx) [inline DDR4 OC Guide.md:342]:
```suggestion
* Geardown mode (GDM) is automatically enabled above 2666MHz, which forces even tCL, even tCWL, even tRTP, even tWR and CR 1T. If you want to run odd tCL, disable GDM. If you're unstable try running CR 2T, but that may negate the performance gain from dropping tCL.
```

File: DDR4 OC Guide.md [hunk 3]
```diff
@@ -579,14 +580,15 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
   * Thanks to Bored and Muren for finding and verifying this on their Asrock motherboards.
 
 ### AMD
-* Try playing around with ProcODT if you can't boot. On Ryzen 1000 and 2000, you should try values between 40Ω and 68.6Ω.  
-On Ryzen 3000, [1usmus](https://www.overclock.net/forum/13-amd-general/1640919-new-dram-calculator-ryzena-1-5-1-overclocking-dram-am4-membench-0-7-dram-bench-480.html#post28049664) suggests 28Ω - 40Ω.  
+* Try playing around with ProcODT if you can't boot. This setting determines the impedence of the On-Die-Termination. According to [Micron](https://www.micron.com/support/~/media/D546161C2C6140BCB0BAEE954AA53433.pdf), higher settings of ProcODT can lead to more stable RAM, but trade off potentially needing higher voltages. On Ryzen 1000 and 2000, you should try values between 40Ω and 68.6Ω due to the considerably weaker memory controller.
```
Comment (integralfx) [inline DDR4 OC Guide.md:583]:
```suggestion
* Try playing around with ProcODT if you can't boot. This setting determines the processor on-die termination impedance. According to [Micron](https://www.micron.com/support/~/media/D546161C2C6140BCB0BAEE954AA53433.pdf), higher settings of ProcODT can lead to more stable RAM, but trade off potentially needing higher voltages. On Ryzen 1000 and 2000, you should try values between 40Ω and 68.6Ω due to the considerably weaker memory controller. 
```

> you should try values between 40Ω and 68.6Ω due to the considerably weaker memory controller

Source? Sure the IMC is weaker but that isn't necessarily related to needing higher ProcODT. Zen 2 can run ProcODT in that range but it has a stronger IMC.

**Review by integralfx — CHANGES_REQUESTED** [review]

File: DDR4 OC Guide.md (hunk 3 above)
Comment (Arshia1381) [reply DDR4 OC Guide.md:583]: this mainly comes down to the aforementioned micron spec. According to that, higher ODT directly leads to higher signal integrity. Furthermore, a weaker IO die and IMC would have worse signal integrity and would be more susceptible to reflections. I'm not claiming a weaker IMC allows for higher ODT, you're right in saying Zen2 can run at high ODT, but the point is that since the memory subsystem for Zen2 and Zen3 is better, it does not require as high of ODT to compensate.

Comment (Arshia1381) [reply DDR4 OC Guide.md]: I could rephrase it to memory subsystem instead of IMC if you'd like. btw, considering you trust Yuri, and source him, note that this is something him and I have discussed before.

File: DDR4 OC Guide.md [hunk 4]
```diff
@@ -579,14 +580,15 @@ The default value is fixed 1.100V and AMD recommends keeping it at that level. I
   * Thanks to Bored and Muren for finding and verifying this on their Asrock motherboards.
 
 ### AMD
-* Try playing around with ProcODT if you can't boot. On Ryzen 1000 and 2000, you should try values between 40Ω and 68.6Ω.  
-On Ryzen 3000, [1usmus](https://www.overclock.net/forum/13-amd-general/1640919-new-dram-calculator-ryzena-1-5-1-overclocking-dram-am4-membench-0-7-dram-bench-480.html#post28049664) suggests 28Ω - 40Ω.  
+* Try playing around with ProcODT if you can't boot. This setting determines the processor on-die termination impedance. According to [Micron](https://www.micron.com/support/~/media/D546161C2C6140BCB0BAEE954AA53433.pdf), higher settings of ProcODT can lead to more stable RAM, but trade off potentially needing higher voltages. On Ryzen 1000 and 2000, you should try values between 40Ω and 68.6Ω due to the considerably weaker memory controller. 
+On Ryzen 3000 and 5000, [1usmus](https://www.overclock.net/forum/13-amd-general/1640919-new-dram-calculator-ryzena-1-5-1-overclocking-dram-am4-membench-0-7-dram-bench-480.html#post28049664) suggests 28Ω - 40Ω, with lower settings being harder to run but potentially helping with voltage requirements.
```
Comment (integralfx) [inline DDR4 OC Guide.md:584]:
```suggestion
On Ryzen 3000 and 5000, [1usmus](https://www.overclock.net/forum/13-amd-general/1640919-new-dram-calculator-ryzena-1-5-1-overclocking-dram-am4-membench-0-7-dram-bench-480.html#post28049664) suggests 28Ω - 40Ω. Lower settings may be harder to run but potentially helps with voltage requirements.
```

Depends on the IC, but Micron Rev E can run 28-32ohms at 5000MHz (confirmed by coffee as well)

Comment (Arshia1381) [reply DDR4 OC Guide.md:584]: definitely but that may be board dependant.

**Review by integralfx — APPROVED** [review]
